tests/integration/test_catboost_model_first_evaluate/model/model_config.xml

```diff
@@ -0,0 +1,8 @@
+<models>
+    <model>
+        <type>catboost</type>
+        <name>titanic</name>
+        <path>/etc/clickhouse-server/model/model.bin</path>
```

Comment:
Btw, where is `model.bin` file in this PR?

Reply:
It disappeared somehow, don't know why. Will add it.